triggers/service/BUILD.bazel

```diff
@@ -94,11 +99,12 @@ da_scala_test(
         "@maven//:ch_qos_logback_logback_classic",
         "@maven//:com_typesafe_akka_akka_actor_typed_2_12",
         "@maven//:com_typesafe_akka_akka_http_core_2_12",
+        "@maven//:eu_rekawek_toxiproxy_toxiproxy_java_2_1_3",
         "@maven//:io_spray_spray_json_2_12",
         "@maven//:org_scalatest_scalatest_2_12",
         "@maven//:org_scalaz_scalaz_core_2_12",
     ],
-)
+) if not is_windows else None  # For now while we don't have Toxiproxy support on Windows.
```

Comment:
Is Toxiproxy not supported on Windows at all, or just in this build setup? If it's the former then it seems like we'd definitely want to split up the tests.

Reply:
It's supported.